app/schemas/models/course_instance.schema.coffee

```diff
@@ -19,6 +19,7 @@ _.extend CourseInstanceSchema.properties,
     language: {type: 'string', 'enum': ['python', 'javascript']}
   hourOfCode: { type: 'boolean', description: 'Deprecated, do not use.' }
   stats: c.object({ additionalProperties: true })
+  startLockedLevel: c.shortString()
```

Comment:
Nice! Sometimes it's good to add a `description` to document what this property does. Not critical though.